monitoring/notification/dashboard.py

```diff
@@ -0,0 +1,567 @@
+            '})',
+        ]),
+    )],
+    thresholdType='absolute',
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+processors_up_current = StateTimeline(
+    title='Notification Processors',
+    description='Active notification processor instances',
+    dataSource='${DS_PROMETHEUS}',
+    legendDisplayMode='hidden',
+    showValue='never',
+    targets=[Target(
+        expr='\n'.join([
+            'label_replace(',
+            'sum by(job) (up{',
+            '    namespace="${namespace}",',
+            '    job=~"${notification_processor}"',
+            '}), "job", "$1", "job", "' + LABEL_EXPR['notification-processors'] +'")',
+        ]),
+        legendFormat='{{job}}',
+    )],
+    thresholdType='absolute',
+    thresholds=[
+        Threshold('red', 0, 0.),
+        Threshold('green', 1, 1.),
+    ])
+
+zookeeper_quorum = Stat(
+    title='Zookeeper Quorum',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='background',
+    noValue='0',
+    reduceCalc='last',
+    targets=[Target(
+        expr='\n'.join([
+            'max(',
+            'quorum_size{',
+            '  namespace="${namespace}",',
+            '  job="${job_zookeeper}"',
+            '})',
+        ]),
+    )],
+    thresholdType='absolute',
+    thresholds=[
+        Threshold('green', 1, 0.),
+    ])
+
+kafka_brokers = Stat(
+    title='Kafka Brokers',
+    dataSource='${DS_PROMETHEUS}',
+    colorMode='background',
+    noValue='0',
+    reduceCalc='last',
+    targets=[Target(
+        expr='\n'.join([
+            'count(',
+            'kafka_server_replicamanager_leadercount{',
+            ' namespace="${namespace}",',
+            ' job="${job_kafka}"',
+            '})',
+        ]),
+    )],
+    thresholdType='absolute',
+    thresholds=[
+        Threshold('green', 1, 0.),
+    ])
+
+kafka_lag = TimeSeries(
+    title='Notification Topics Lag',
+    description='Consumer lag of each of the notification topics',
+    dataSource='${DS_PROMETHEUS}',
+    gradientMode='opacity',
+    lineInterpolation='smooth',
+    fillOpacity=10,
+    spanNulls=True,
+    legendDisplayMode='table',
+    legendPlacement='right',
+    unit=UNITS.SECONDS,
+    targets=[Target(
+        expr='\n'.join([
+            'label_replace(',
+            'max by(topic) (',
+            '    kafka_consumergroup_group_lag_seconds{'
+            '       topic=~"${notification_topics}"',
+            '}), "topic", "$1", "topic", "' + LABEL_EXPR['topics'] +'")',
+        ]),
+        legendFormat='{{topic}}',
+    )])
+
+success_rate = GaugePanel(
+    title='Success Rate',
+    dataSource='${DS_PROMETHEUS}',
+    min=0,
+    max=1,
+    noValue='-',
+    format=UNITS.PERCENT_UNIT,
+    targets=[Target(
+        expr='\n'.join([
+            'sum(rate(',
+            ' ' + Metrics.DELIVERY_LAG.count(),
+            ')) /',
+            'sum(rate(',
+            ' ' + Metrics.DELIVERY_LAG.count('status=~".*"'),
```

Comment:
it may be faster (in grafana/prometheus) to do it the other way: e.g. instead of asking grafana/prometheus to filter "all values" with regex, specifying no 'status' label will pick up everything...

that would mean removing the default value for DELIVERY_LAG's status label, and give the following expression:

```
expr = '\n'.join([
   'sum(rate(' + Metrics.DELIVERY_LAG.count(status='success') + '))',
   '/',
   'sum(rate(' + Metrics.DELIVERY_LAG.count() + '))',
])
```